client/devdocs/design/index.jsx

```diff
@@ -256,6 +257,7 @@ export default class DesignAssets extends Component {
 					<StepProgress readmeFilePath="step-progress" />
 					<Suggestions readmeFilePath="/packages/components/src/suggestions" />
 					<SuggestionSearchExample />
+					<SummaryButtonExample />
```

Comment:
@mirka correct me if I'm wrong, but I think we're soon going to remove the dev docs, in favour of Storybook (and soon a reference site)? Should we just have Storybook stories for now?

Reply:
I read about it somewhere, but I don't think it hurts to add until then, does it?

Reply:
For every devdoc example in here, a human will need to grok it and verify whether it matches 1:1 with any existing SB stories for the component, decide how to resolve any discrepancies, and remove the file. So yes, it'd be better if we don't add to that task list 😅

Reply:
Sounds good. I removed it.